zulip_bots/zulip_bots/lib.py

```diff
@@ -9,7 +9,7 @@
 
 
 from typing import Any, Optional, List, Dict, IO, Text
-
+from typing_extensions import Protocol
```

Comment:
This module has not been added to dependencies to be installed?

Reply:
I don't think we need a separate installation for this library.

Reply:
I don't believe typing_extensions is included in python3.6?
I tested this in a venv and `zulip-run-bot` rightly says
```
ModuleNotFoundError: No module named 'typing_extensions'

Reply:
Will add that.